apps/resources/src/components/CollectionsPage/CollectionsVideoContent/Questions/Questions.tsx

```
@@ -0,0 +1,90 @@
+
+  return (
+    <div className="xl:w-2/5">
+      <div className="questions-block pt-16 xl:pt-4">
+        <div className="padded mb-6 flex flex-wrap items-center justify-between">
+          <h4 className="flex flex-shrink-0 items-center gap-4 py-4 text-sm font-semibold tracking-wider text-red-100/70 uppercase xl:text-base 2xl:text-lg">
+            {questionsTitle}
+          </h4>
+
+          <a
+            href="https://issuesiface.com/talk?utm_source=jesusfilm-watch"
+            target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)